kong/clustering/services/sync/rpc.lua

```diff
@@ -54,13 +54,30 @@ local function full_sync_result()
 end
 
 
+local function get_current_version()
+  return declarative.get_current_hash() or DECLARATIVE_EMPTY_CONFIG_HASH
+end
+
+
+local is_empty_version
+do
+  local byte = string.byte
+  local CHAR_V = byte("V")
+
+  -- version string must start with char 'V'
+  is_empty_version = function(v)
```

Comment:
The name is misleading. It could be an incorrect version string

Reply:
done, thanks.